apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -844,6 +844,29 @@ const PoolCandidatesTable = ({
         ),
       },
     ),
+    columnHelper.accessor(
+      ({
+        poolCandidate: {
+          user: { lookingForEnglish, lookingForFrench, lookingForBilingual },
+        },
+      }) => {
+        const arr = [];
+        if (lookingForEnglish) {
+          arr.push(intl.formatMessage(commonMessages.english));
+        }
+        if (lookingForFrench) {
+          arr.push(intl.formatMessage(commonMessages.french));
+        }
+        if (lookingForBilingual) {
+          arr.push(intl.formatMessage(commonMessages.bilingualEnglishFrench));
+        }
+        return arr.join(", ");
+      },
+      {
+        id: "languageAbility",
+        header: intl.formatMessage(commonMessages.workingLanguageAbility),
+      },
+    ),
```

Comment:
Could we pull this out into a utility file that can be used in both the PoolCandidatesTable and UserTable?